src/Transports.AspNetCore/WebSockets/GraphQLWs/PingPayload.cs

```diff
@@ -0,0 +1,12 @@
+namespace GraphQL.Server.Transports.AspNetCore.WebSockets.GraphQLWs;
+
+/// <summary>
+/// The payload of the ping message.
+/// </summary>
+public class PingPayload
+{
+    /// <summary>
+    /// The unique identifier of the ping message.
+    /// </summary>
+    public string? id { get; set; }
```

Comment:
In order to serialize in camel case, we need to define the property in camel case here.  However, this property (including its name, capitalization and contents) are purely arbitrary.  So we could use pascal case -- but it would be serialized as such.

Using attributes is not an option because GraphQL supports two serialization libraries (System.Text.Json and Newtonsoft.Json), neither of which are referenced here (e.g. for .NET 4.8).